src/test/java/pom/DeletePlaylistPage.java

```diff
@@ -0,0 +1,30 @@
+package pom;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class DeletePlaylistPage extends BasePage{
+
+    public DeletePlaylistPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void openPlaylist() {
+        WebElement emptyPlaylist = findWebElement(By.cssSelector(".playlist:nth-child(3)"));
+        clickToElement(emptyPlaylist);
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)